src/main.rs

```diff
@@ -56,34 +59,73 @@ fn main() {
                 // Can this ever happen?
                 Ok(())
             } else {
-                println!("Fuzzing found errors!");
+                if let Some(ref mut terminal) = term_stdout {
+                    let _ = terminal.fg(term::color::YELLOW);
+                    println!("Fuzzing found errors!");
+                    let _ = terminal.reset();
+                } else {
+                    println!("Fuzzing found errors!");
+                }
```

Comment:
@frewsxcv @Manishearth @nagisa So, are you ok with adding the `term` code to a new module, say `utils` or leaving it into `lib` ?